website/docs/patterns/button-alignment/index.md

```diff
@@ -2,7 +2,7 @@
 title: Button alignment
 description: Guidelines for aligning, ordering, and grouping action buttons.
 navigation:
-  hidden: true
+  hidden: false
```

Comment:
suggestion: instead of `false` let's just get rid of the `hidden` argument (and at that point also the `navigation` one)

Reply:
Hmmmm, that's a good point, I didn't think about that since the content was merged already. 

I'll remove the hidden for now. For content review purposes I'm not sure what makes sense, but should I remove the content in a new PR and add it back in a subsequent PR?

Reply:
No idea :) 

@alex-ju git guru, any ideas/suggestion how we can resolve this?

Reply:
@jorytindall nne option would be (@alex-ju what do you think?) to do two sequential commits in this PR, one in which you delete the markdown files for the documentation and one in which you reverse the commit (essentially adding back the files). In this way we could comment on the "revert" commit directly (it's possible, afaik).
At the end of the process, we squash all the commits in a single one, and it's like nothing has happened.

Reply:
Let me see if I got this right. The documentation was added in #1092, but was hidden at the time. In this PR we make it visible and remove some exceptions. However we'd like to re-review the content, so a visual diff in GitHub would be necessary.

First of all, merging the branch 'main' into jt/review-button-alignment didn't help as it blocked us from being able to rebase. So I got rid of that merge first so that we can operate on this branch. This means you'd have to reset the head before operating further on this branch, Jory. So please run the following command on this branch, on your local machine, @jorytindall: `git reset --hard origin/jt/review-button-alignment` – this will make your local setup match what you see now in GitHub and will allow you to add further commits as needed based on feedback.

Didoo's suggestion sounds good to me. So this is what I did.
The original PR had 16 commits, so quite difficult to revert those, but we can use the merge commit instead (which sums up the changes). Because it is a merge commit it requires some special attention, so I did `git revert 675e257 -m 1` (if it wasn't a regular commit `git revert 675e257` would've been enough). Then I reverted the revert commit so that we get the diff we wanted and reviewers can now make suggestions on the content.

Hope that helps.

Reply:
Whoa, super helpful, thanks @alex-ju !